service/src/main/java/com/fit_goal/util/MailSender.java

```diff
@@ -0,0 +1,31 @@
+package com.fit_goal.util;
+
+import lombok.experimental.UtilityClass;
+import org.simplejavamail.email.Email;
+import org.simplejavamail.email.EmailBuilder;
+import org.simplejavamail.mailer.Mailer;
+import org.simplejavamail.mailer.MailerBuilder;
+
+@UtilityClass
+public class MailSender {
+
+    //TODO configure in config.yml
+    private final String HOST = "smtp.mailtrap.io";
```

Comment:
these configs should be in config.yml

Reply:
please refer to ExecutorServiceConfig in caching service as an example

Reply:
fixed